projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -81,13 +82,17 @@ def configure_initial_logging_before_anything():
     3. When initialize_job hook (applicable for vdk run only) is executed then below configure_loggers function
         is run which adds more context to the logs and initializes syslog handler (if configured to do so)
     """
-    log_level = "WARNING"
-    if os.environ.get(LOG_LEVEL_VDK, None):
-        log_level = os.environ.get(LOG_LEVEL_VDK)
-    elif os.environ.get("VDK_LOG_LEVEL_VDK", None):
-        log_level = os.environ.get("VDK_LOG_LEVEL_VDK")
+    warnings.warn(
```

Comment:
I don't think we need this until we've released the plugin.

Reply:
ok, removed it.